### DISCUSSION TITLE
ESC Mania

### DISCUSSION THREAD

**Discussion by dannyjacobs** [dannyjacobs/ECHO#47] [General]
The Mcgill group reports difficulty finding decent ESCs. Heres what I think we know so far:

1. The [2018 design](https://github.com/dannyjacobs/ECHO/blob/master/hardware/Chiropter%20BoM%20-%20Revised%20Build%20Sheet.pdf) used the [DJI E800 kit](https://www.dji.com/e800) which came in a kit with 350kV motors and _620S ESC_s. 
2. These kits are out of stock (eg [here](https://www.bhphotovideo.com/c/product/1141068-REG/dji_cp_ep_000025_e800_tuned_propulsion_system.html) or [here](https://www.powerhobby.com/dji-e800-x4-tuned-propulsion-system-quadcopter-620s-esc-3510-motor-1345-props.html))
3. Its not clear if DJI will continue to make such a kit.  Subsequent offerings seem to be mainly replacement parts for their pro drones. Anecdotally these have been difficult to obtain and easily confused by suppliers with replacement parts for other (smaller) DJI drones.
4. The E800 specs are what we want to replicate with any new builds.

**Specifications**
Max Thrust	2100 g/rotor @ 25 V Sea Level
Takeoff Weight Recommended	800 g/rotor
Battery Recommended	6S LiPo
Working Temperature	-10°C ~ 40°C
**Motor**
Stator Size	35×10 mm
KV	350 rpm/V
Weight	106 g
**ESC**
Max Allowable Voltage	26 V
Max Allowable Current (Persistent)	20 A
Signal Frequency	30 Hz ~ 450 Hz
Battery	3 S ~ 6 S LiPo
Size	74x32x10 mm
Cable length	420 mm
Weight (with Cable)	43 g
Weight (without Cable)	30 g
**Propeller**
Diameter / Thread Pitch	13×4.5 inch
Weight	19 g

5. The closest DJI product is the E600 series. The number in the name is the single motor thrust.  The 600 delivers 200g lower thrust. It also comes with a shorter prop and spins faster. I can find one[ in stock somewhere](https://www.kopterworx.com/dji-e600-tuned-propulsion-system-6-motors.html), so theres an existence proof.
6. An interesting alternative ESC might be the Eflight V2 40A. New in 2020, and [in stock](https://store.flitetest.com/eflite-40amp-pro-switchmode-bec-brushless-esc-v2-ec3-efla1040b/p264105)

**Comment by hsinc** [comment]
The most recent ESC we've tried ordering is the DJI 620D: https://store.dji.com/ca/product/matrice-100-esc-kit.  Shortly after we placed the order, the website changed to report that the parts were no longer in stock.  Our parts still haven't shipped yet, so it's unclear what's going to happen.  The other things that we've purchased (but have yet to try) are these motor/ESC combinations: https://www.ebay.ca/itm/DJI-Inspire-1-V2-0-Pro-Part-3510H-Motor-ESC-CW-M2-M4-US-Dealer/202572896903.  If those work, they're all over the place on ebay and very easy to get a hold of.

**Reply by dannyjacobs** [reply]
Have you found any official specs for the 3510Hs?  Current/voltage rating for the ESC and kV for the motor?  Google likes[ this description](https://tienda.stockrc.com/epages/eb0140.sf/en_US/?ObjectPath=/Shops/eb0140/Products/%222020-P1-BE-E800%20MOTOR%20CCE%22) which suggests that 3510H is the same as the E800.  Also I observe that the stator size is 35mm x 10mm. Suggests the part name is genericly referring to the size

**Reply by harrylerman** [reply]
I think the kV rating is different, 420 kV for the 3510H vs 350 for the 3510/E800.

The part number corresponding to the size of the motor seems similar to how the prop numbering works, ex how the 1345 props are 13 inches length with a 4.5 inch pitch.

**Reply by hsinc** [reply]
Forgot to mention that the 620D ESCs also appear to be available here: https://www.bhphotovideo.com/c/product/1185301-REG/dji_cp_tp_000027_esc_kit_for_matrice.html.  But they're listed as special order with a lead time of 6-10 weeks, which I suppose isn't inconsistent with the order that we placed directly with DJI about a month ago that has yet to ship.

**Reply by hsinc** [reply]
Well...2+ months later, DJI finally contacted me to let me know that the 620D ESCs are EOL'd.  I've cancelled the order.  But since the ebay-style motor/ESC combo seems to be working so far, that might be a viable path forward for the future.

**Comment by hsinc** [comment]
I've also looked at the T-motor propulsion kits: https://store-en.tmotor.com/category.php?id=13.  None of them seem to be a great match for the E800 kit, and I haven't spent much time playing with alternate designs on ecalc.

**Reply by dannyjacobs** [reply]
Arg, what is it with carbon fiber props and tiny little screws??  Lots of money for not much gain.  The DJI plastic props pop on with a quarter turn.  Lifes too short for anything less. Looks like the 3510s DJI motors are still the best thing going. Those ESCs you got are close.  Annoying to have to make an adaptor board.

**Reply by dannyjacobs** [reply]
The [air gear 450s](https://store-en.tmotor.com/goods.php?id=726) have the good simple assembly and cheap props, but are quite a bit lower power. 1.2kg thrust vs 2.1kg on the DJI. This is probably mostly due to the smaller diameter motor. Tmotor outer diameter is 22mm, DJI stator (the stationary bit with the windings) is 35mm.

PS Tmotor catalog now includes motors for "manned aircraft"!  Including a beast with a 153mm stator. Uses 30kW to drive a 67inch prop for 100kg of lift. So we can always go bigger, apparently.

**Reply by eegan** [reply]
One thing that _isn't_ all over the web is the PWM connector pinout for those ESC / motor combos. Hopefully we'll reverse-engineer it within the next week or so.
